dcpy/lifecycle/builds/dbt.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+import typer
+from ruamel.yaml import YAML
+
+from dcpy.utils.logging import logger
+from dcpy.models.lifecycle.builds import InputDataset, Recipe
+from dcpy.lifecycle.builds import plan
+
+DEFAULT_RECIPE = "recipe.yml"
+
+app = typer.Typer(add_completion=False)
+
+yaml = YAML()
```

Comment:
Am I annoyed that this package requires that you instantiate a YAML object to read/write yaml from files? Yes I am